sql/parse/parse.go

```diff
@@ -702,6 +708,22 @@ func convertDDL(ctx *sql.Context, query string, c *sqlparser.DDL) (sql.Node, err
 	}
 }
 
+func convertMultiAlterDDL(ctx *sql.Context, query string, c *sqlparser.MultiAlterDDL) (sql.Node, error) {
+	statementsLen := len(c.Statements)
+	if statementsLen == 1 {
+		return convertDDL(ctx, query, c.Statements[0])
+	}
+	statements := make([]sql.Node, statementsLen)
+	var err error
+	for i := 0; i < statementsLen; i++ {
+		statements[i], err = convertDDL(ctx, query, c.Statements[i])
+		if err != nil {
+			return nil, err
+		}
+	}
+	return plan.NewBlock(statements), nil
```

Comment:
This doesn't match the structure in ast.go. What happened to table name?

Reply:
Think you might need a new kind of node here, not just a Block

Reply:
The table name at the root is mainly for display purposes. Each DDL statement in the slice is self contained. It’s why I chose a block rather than a new type, since we could just iterate over the pre-existing DDL code rather than making a new type that way.

Reply:
OK, I get it now. Carry on.